face_track/face_track.py

```diff
@@ -19,12 +19,15 @@
 
 import time
 
+import datetime
+
 import rospy
 import tf
 import math
 import logging
 
 from std_msgs.msg import Int32, String
+from chatbot.msg import ChatMessage
```

Comment:
please stop importing ChatMessage.   This has been an issue that has been biting us for more tha  a year. I keep getting rid of ChatMessage, it keeps coming back to life. Can we PLEASE kill it forever-more?

This breaks the open-source version of the robot.